src/GuiBase/Viewer/Viewer.cpp

```diff
@@ -247,21 +254,15 @@ void Gui::Viewer::intializeRenderer( Engine::Renderer* renderer ) {
     renderer->lockRendering();
 }
 
-void Gui::Viewer::resizeGL( int width_, int height_ ) {
-    if ( isExposed() )
-    {
-        // Renderer should have been locked by previous events.
-        m_context->makeCurrent( this );
-
-        // see issue #261 Qt Event order and default viewport management (Viewer.cpp)
-        // https://github.com/STORM-IRIT/Radium-Engine/issues/261
-#ifndef OS_MACOS
```

Comment:
Are these changes validated on MacOS ?

Reply:
Not able to test, there is a dylib problem with assimp. I'll try to fix it.

Reply:
Using DYLD_LIBRARY_PATH, added to README.md